.github/workflows/override_api_endpoint-test.yml

```diff
@@ -20,8 +20,7 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - version: '2.5.0'
-          - version: '3.3'
+          - version: '3.13.7'
```

Comment:
I prefer having the newest PATCH e.g. `3.13.x`

Unfortunately it doesn't work for `dart-lang/setup-dart` action.

Reply:
I think it works now https://github.com/dart-lang/setup-dart/pull/106/files

Reply:
oh wow that's awesome